src/borg/testsuite/archiver/diff_cmd.py

```diff
@@ -170,14 +172,17 @@ def get_changes(filename, data):
                 assert {"type": "removed link"} in get_changes("input/link_removed", joutput)
 
                 # Symlink replacing or being replaced
-                assert any(
-                    chg["type"] == "mode" and chg["new_mode"].startswith("l")
-                    for chg in get_changes("input/dir_replaced_with_link", joutput)
-                )
-                assert any(
-                    chg["type"] == "mode" and chg["old_mode"].startswith("l")
-                    for chg in get_changes("input/link_replaced_by_file", joutput)
-                )
+
+                # TODO: Add mode test flag to assert function
```

Comment:
What exactly do you mean by that?

Reply:
This was to be removed before the second commit. Will remove.

Reply:
Amended last commit.